### PR TITLE
[chore] disable gosec checks that cause performance issues

### PR DESCRIPTION
**Pull request by codeboten**
Same as https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33192

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -119,6 +119,13 @@ linters-settings:
           files:
             - "!**/*_test.go"
 
+  gosec:
+    excludes:
+      # https://github.com/golangci/golangci-lint/issues/4735
+      - G601
+      # https://github.com/golangci/golangci-lint/issues/4735
+      - G113
```
Comment (bogdandrutu) [inline .golangci.yml:127]: Please create an issue on our side to remove this excludes later.

**Review by bogdandrutu — APPROVED** [review]

**Review by songy23 — APPROVED** [review]

File: .golangci.yml [hunk 2]
```diff
@@ -119,6 +119,13 @@ linters-settings:
           files:
             - "!**/*_test.go"
 
+  gosec:
+    excludes:
+      # https://github.com/golangci/golangci-lint/issues/4735
```
Comment (codeboten) [inline .golangci.yml:124]:
```suggestion
      # https://github.com/open-telemetry/opentelemetry-collector/issues/10213
```

File: .golangci.yml [hunk 3]
```diff
@@ -119,6 +119,13 @@ linters-settings:
...
+      - G601
+      # https://github.com/golangci/golangci-lint/issues/4735
```
Comment (codeboten) [inline .golangci.yml:126]:
```suggestion
      # https://github.com/open-telemetry/opentelemetry-collector/issues/10213
```

File: .golangci.yml (hunk 1 above)
Comment (codeboten) [reply .golangci.yml]: Updated comment with issue in this repo

**Pull request merged**